Changelog.rst

```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+v2.0.0
+------
+
+* Dry run behavior is now activated by default. We start by computing all the changes and then ask if they look good before doing anything. This also means we no
+  longer need to pause right before calling ``git push``. Consequently, the ``--dry-run`` option is gone.
```

Comment:
the dry-run could be useful, by not asking the question at all and exiting immediately. bike shedding ?